lightning/src/ln/channel.rs

```diff
@@ -449,6 +475,11 @@ pub(super) struct Channel<Signer: Sign> {
 	commitment_secrets: CounterpartyCommitmentSecrets,
 
 	channel_update_status: ChannelUpdateStatus,
+	/// Once we reach `closing_negotiation_ready`, we set this, indicating if closing_signed does
+	/// not complete within a single timer tick (one minte), we should force-close the channel.
+	/// Note that this field is reset to false on deserialization to give us a chance to connect to
+	/// our peer and start the closing_signed negotiation fresh.
```

Comment:
Add rational, "Beyond the one-minute window, we expect feerate to fluctuate and as such negotiation to become obsolete"?